Patch-Source: https://github.com/AyatanaIndicators/ayatana-indicator-printers/commit/b4e36f43ec5ef68b05679a21c3a67ccc811034eb

From b4e36f43ec5ef68b05679a21c3a67ccc811034eb Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Tue, 7 Dec 2021 21:47:00 +0100
Subject: [PATCH] Remove bash-ism from configure.ac

The += operator is not available e.g. in busybox ash. Instead of trying
to extend CUPS_CFLAGS and _LIBS, we can just set them as nothing sets
them beforehand.
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0986f8c..8919cfd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -50,8 +50,8 @@ if test "x$CUPS_CONFIG" = "xno"; then
     AC_MSG_ERROR([could not find cups-config])
 fi
 AC_CHECK_HEADER(cups/cups.h,,AC_MSG_ERROR([could not find cups.h]))
-CUPS_CFLAGS+=`$CUPS_CONFIG --cflags`
-CUPS_LIBS+=`$CUPS_CONFIG --libs`
+CUPS_CFLAGS=`$CUPS_CONFIG --cflags`
+CUPS_LIBS=`$CUPS_CONFIG --libs`
 SERVICE_CFLAGS="$SERVICE_CFLAGS $CUPS_CFLAGS"
 SERVICE_LIBS="$SERVICE_LIBS $CUPS_LIBS"
 
-- 
2.34.1

